src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const string = sourceString.split(';');
```

Comment:
Consider adding validation to check if `sourceString` is not empty or malformed before splitting. This will help prevent potential errors when processing the string.